dist/ti-modal.js

```diff
@@ -144,3 +83,78 @@ define("ti-modal/initializer",
       });
     });
   });
+define("templates/ti-modal", function () { return Ember.TEMPLATES["ti-modal"] = Ember.Handlebars.template(function anonymous(Handlebars,depth0,helpers,partials,data
```

Comment:
I think this is wrong, needs to be `define("templates/ti-modal", ["exports"], function (__exports__) { __exports__["default"] = Ember.Handlebars.template(...`